draft-ietf-quic-transport.md

```diff
@@ -205,28 +205,28 @@ Out-of-order packet:
 
 Endpoint:
 
-: An entity that can participate in a QUIC connection by generating,
-  receiving, and processing QUIC packets. There are only two types of endpoint
-  in QUIC: client and server.
+: An entity that can participate in a QUIC connection by generating, receiving,
+  and processing QUIC packets. There are only two types of endpoint in QUIC:
+  client and server.
 
 Client:
 
 : The endpoint initiating a QUIC connection.
 
 Server:
 
-: The endpoint accepting incoming QUIC connections.
+: The endpoint accepting an incoming QUIC connection.
```

Comment:
👎 : the proposed version sounds worse.

Reply:
Can you expand on why?  I'm adjusting it because nothing in the server role requires it to be accepting multiple connections.  We speak about clients initiating "a QUIC connection," and for symmetry should talk about a server accepting an individual QUIC connection.  Dealing with multiple simultaneous connections is something clients and servers both might do, but it's not fundamental to the definition of either client or server.

Reply:
You are right: nothing in the server role mandates that the server accept more than one connection.  On the other hand, in the majority of cases, from literature to practical implementations, a server does accept more than one connection.  Hence, my own mental model is that of "one server, many clients."  I predict this model is shared by the majority of readers, which is why I dislike the proposed change in wording.

This is not to say that the new version is bad; it is just worse in comparison.  The former definition causes less mental load because it fits the mental model.

While we are on this sentence: perhaps drop "incoming?"

Reply:
```suggestion
: The endpoint that accepts a QUIC connection.
```